ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml

```diff
@@ -73,6 +73,19 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-ibmcloud-ipi-ovn
   timeout: 5h0m0s
+- as: data-path-9nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: ibmcloud-qe
+    env:
+      SET_ENV_BY_PLATFORM: custom
+      WORKERS: "3"
+    test:
+    - ref: cucushift-installer-wait
+    - ref: openshift-qe-perfscale-ibm-data-path-sg
```

Comment:
need change the order?

Reply:
but openshift-qe-perfscale-ibm-data-path-sg will fail and I don't want to run network tests before checking if the rules are added.... but the cucushift is failing with imagepullbackoff

Reply:
The code is working fine I could see the rules added to the --wide security group. I have taken off the debug "sleep" code off now.
currently it failed for (runtime error: invalid memory address or nil pointer dereference)}

@liqcui  suggested to increase timeout to 8 hours. Please review

Reply:
you can add the cucushift-installer-wait step, and set SLEEP_DURATION <20h
when stop debugging, delete the cucushift-installer-wait pod, the post steps will be run